frontend/magic_links/templates/signed_out.html

```diff
@@ -26,11 +26,16 @@ <h1 class="govuk-heading-xl">
     </p>
 {% endif %}
 
+{% if fund and round %}
 <p class="govuk-body">{% trans %}To re-access your account, please click below to request a new link.{% endtrans %}</p>
 
 {{ govukButton({
     "isStartButton": false,
     "href" : url_for('magic_links_bp.new', fund=fund, round=round),
     "text": gettext("Request a new link")
 }) }}
+{% else %}
+<p class="govuk-body">{% trans %}To re-access your account, please use the link originally provided in the email sent you.{% endtrans %}</p>
```

Comment:
Not sure this would work, as magic links are invalid after claiming them - should we ask them to request a new magic link, and provide a link to where to do so?

Reply:
ahh this link is actually referring to the link we send them originally inviting them to apply, which will take them to frontend and then through the magic link flow so all good 👍🏾